RedPitaya_emulator/srcbin/emul.c

```diff
@@ -0,0 +1,110 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<stdint.h>
+#include<time.h>
+#include<signal.h>
+
+#include "TCP_API.h"
+
+#define PORT 7538
+
+float x0=80.0;
```

Comment:
I think we should define and read a config file for probe settings associated with each images